src/services/fetchers/nacion.ts

```diff
@@ -21,15 +21,17 @@ class Nacion {
     
       return {
         name: 'Nación',
+        url: "https://bit.ly/2FWkJE4"
```

Comment:
missing `,` at the end

Reply:
We need to update the URL to point here https://hb.redlink.com.ar/bna/login.htm

Reply:
Done